ext/crypto/lib.deno_crypto.d.ts

```diff
@@ -326,6 +326,7 @@ declare interface Crypto {
       | Uint16Array
       | Uint32Array
       | Uint8ClampedArray
+      | BigUint64Array
```

Comment:
You missed `BigInt64Array`

Reply:
![image](https://user-images.githubusercontent.com/22394081/170097561-a278bb85-c5dd-4b56-b282-27bf01a9f2b5.png)

Spec mentions that `getRandomValues` only takes integer-based `TypedArray`, does removing `Float64Array` and `Float32Array` can be done in this PR itself?

Reply:
Good catch! Sure, we can apply these changes. What do you think? @lucacasonato

Reply:
SGTM

Reply:
Should be fixed now